packages/pharos-site/src/components/statics/component-status/StatusTable.tsx

```diff
@@ -61,6 +79,7 @@ const StatusTable: FC = () => {
     });
 
     setStateTable(allComponentStatuses);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Adding `ref` to the dependency would cause a re-render loop eventual leading to the `maximum update depth exceeded` error